jdks.md

```diff
@@ -83,8 +83,28 @@ Next to this here is a table with direct links to all current versions:
 | macOS / x64   | 8u172        |[download](https://goo.gl/g68cNn) |
 | Windows / x64 | 8u172        |[download](https://goo.gl/tX1wNF) |
 
+Another player is [SAP](https://www.sap.com) that distributes (free OpenJDK builds)[https://sap.github.io/SapMachine/] next to a closed source SAP JVM. 
```

Comment:
Should be \[free OpenJDK builds\](https://sap.github.io/SapMachine/) and not the other way round (otherwise you don't see the link). This is Markdown and not ASCIIDoc :)

If you want you can also add a link for the "closed source SAP JVM":

[closed source SAP JVM](https://tools.hana.ondemand.com/#cloud)

Reply:
fixed in master. THX